Simplified/Views/NYPLRoundedButton.swift

```diff
@@ -139,16 +139,24 @@ private let NYPLRoundedButtonPadding: CGFloat = 6.0
     updateColors()
   }
   
+  private var customAccessibilityLabel: String?
+  
   override var accessibilityLabel: String? {
     get {
+      if let label = customAccessibilityLabel {
+        self.customAccessibilityLabel = nil
+        return label
+      }
       guard !self.iconView.isHidden,
```

Comment:
should we remove the `isHidden` check? so that we always say the time remaining -- Otherwise once the custom label is fetched once, the next time I think it will fall inside the guard and not say the time indication.

Reply:
Removed